README.md

```diff
@@ -77,63 +77,63 @@ This project contains components for Vaadin 20+. Please see individual repositor
-| [`<vaadin-text-field>`](https://github.com/vaadin/web-components/tree/master/packages/text-field)             | [![npm version](https://badgen.net/npm/v/@vaadin/text-field/next)](https://www.npmjs.com/package/@vaadin/text-field)             | [Issues](https://github.com/vaadin/web-components/labels/vaadin-text-field)       |
-| [`<vaadin-time-picker>`](https://github.com/vaadin/web-components/tree/master/packages/time-picker)           | [![npm version](https://badgen.net/npm/v/@vaadin/time-picker/next)](https://www.npmjs.com/package/@vaadin/time-picker)           | [Issues](https://github.com/vaadin/web-components/labels/vaadin-time-picker)      |
-| [`<vaadin-upload>`](https://github.com/vaadin/web-components/tree/master/packages/upload)                     | [![npm version](https://badgen.net/npm/v/@vaadin/upload/next)](https://www.npmjs.com/package/@vaadin/upload)                     | [Issues](https://github.com/vaadin/web-components/labels/vaadin-upload)           |
-| [`<vaadin-virtual-list>`](https://github.com/vaadin/web-components/tree/master/packages/virtual-list)         | [![npm version](https://badgen.net/npm/v/@vaadin/virtual-list/next)](https://www.npmjs.com/package/@vaadin/virtual-list)         | [Issues](https://github.com/vaadin/web-components/labels/vaadin-virtual-list)     |
+| Component                                                                                                       | npm version (latest)                                                                                                          | npm version (next)                                                                                                                 | Issues                                                                             |
+| --------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------- |
+| [`<vaadin-accordion>`](https://github.com/vaadin/web-components/tree/master/packages/accordion)                 | [![npm version](https://badgen.net/npm/v/@vaadin/accordion)](https://www.npmjs.com/package/@vaadin/accordion)                 | [![npm version](https://badgen.net/npm/v/@vaadin/accordion/next)](https://www.npmjs.com/package/@vaadin/accordion)                 | [Issues](https://github.com/vaadin/web-components/labels/vaadin-accordion)         |
```

Comment:
The next version badge also links to the latest version on npm. We can link it to the next tag by adding `/v/next` to the URL, for example for accordion: https://www.npmjs.com/package/@vaadin/accordion/v/next

Reply:
Good catch, fixed.